ArduCopter/mode_auto.cpp

```diff
@@ -506,9 +506,7 @@ void ModeAuto::circle_movetoedge_start(const Location &circle_center, float radi
     copter.circle_nav->set_center(circle_center);
 
     // set circle radius
-    if (!is_zero(radius_m)) {
-        copter.circle_nav->set_radius_cm(radius_m * 100.0f);
-    }
+    copter.circle_nav->set_radius_cm(radius_m * 100.0f);
```

Comment:
I wonder if there's a default radius we should set instead if radius is not > 0?

Reply:
if radius is 0 it should pirouette in place, IMO

Reply:
I think pirouette in place feels like the natural limit as loiter radius -> 0